Change Volunteer/Institutionally Supported tag colour

acrymble opened the pull request:
Closes #1668 - changes the colour of the tags on the Project Team page for "Volunteer" and "institutionally supported" as these are not roles.

Does so by adding a new "Status" tag to the authors.yml file and attempting to copy code for displaying those tags (though I suspect I haven't got that right yet).

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

adamcrymble committed:
add "status" tag for volunteer / supported

adamcrymble committed:
attempt to add status in diff colour

mdlincoln commented:
Thanks for getting this started. I'm afraid i won't have bandwidth in the next week to look, but can someone else from @programminghistorian/technical-team be willing to help adam out in the meantime?

walshbr commented:
I can take a look locally - it looked like something wasn't quite taking right when I looked at the preview just now. I'm in and out of meetings but will try to take a look. Depending on how long it takes I might have to do it tomorrow.

jenniferisasi commented:
Trying to decipher the code to help ... 
Couple of thing that come to mind: 
- @acrymble I remember not every member having a volunteer|inst-supported tag first, right? Have you now added either to everyone? 
- like the role_label part, isn't an `if` loop necessary?

acrymble commented:
@jenniferisasi I haven't added them to everyone, no. I figured that would be easy to do afterwards.

walshbr commented:
Let me take a crack at it @jenniferisasi first - I think I made progress earlier before I got sucked into meetings. I think you're on the right track though.

walshbr committed:
Correcting some syntax

walshbr commented:
I just pushed a thing that I think got at what you were after @acrymble if you want to check out the preview now. You were pretty close in copying the material from the other code snippet - I just had to adjust things a little.

To @jenniferisasi's point - because they're no if statement right now to check to see if a person has this particular label, it will generate it regardless (and that HTML snippet will have content inside it). This HTML winds up being more or less invisible to the user looking at a browser window, but I'm not sure how it would interact with screen readers to have empty HTML tags behind the scenes. Just something to think about.

walshbr committed:
Merge branch 'gh-pages' into issue1668

walshbr commented:
And I just re-merged gh-pages into this branch - I think the tests were failing b/c it was out of sync with our main branch. Anyway, let me know if the current look is what you were after @acrymble and I can approve it.

jenniferisasi commented:
Looks nice! Good job @acrymble and @walshbr! 

I see that only two members don't have that tag; would the issue of "an empty tag" be solved if we populate them? I guess Adam tried to contact them but got no response?

acrymble commented:
Thank you so much @walshbr that's perfect. Do we like the colour?

acrymble commented:
@jenniferisasi yes I had a couple of people not respond, but we can follow up.

mdlincoln commented on _includes/contact-info.html:
```diff
@@ -25,6 +25,11 @@
 List subteams within the larger editorial team
 {% endcomment %}
 <p>
+{% for status in member.status %}
```
This shouldn't need to be a for loop since everyone only has 1 `status`

walshbr replied:
This is why we pay you the big bucks.

acrymble replied:
@walshbr actually we don't pay him at all... ;P

acrymble replied:
This should now be working @mdlincoln . Thanks.

adamcrymble committed:
remove for loop on .status html snippet

adamcrymble committed:
change status to member.status

Removing the for loop around volunteer/institutionally supported tags made everyone have a green "managing editor" tag. I suspect it's because I didn't tell it where to find the status label properly.

rivaquiroga commented:
@acrymble, I asked María José and she should have the volunteer tag

adamcrymble committed:
add member.status to if statement

Otherwise it is defaulting to "managing editor" if no status is found.

adamcrymble committed:
afanador llach to volunteer
